types/workgrid-toolbar/tsconfig.json

```diff
@@ -19,5 +20,5 @@
     "files": [
         "index.d.ts",
         "workgrid-toolbar-tests.ts"
-    ]
+	]
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/workgrid-toolbar/tsconfig.json;
  Line: 23;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.